src/lgr_basic/views.py

```diff
@@ -37,8 +37,8 @@ def form_valid(self, form):
 
         labels_cp = form.cleaned_data['labels']
         labels_file = form.cleaned_data.get('labels_file')
-        rz_lgr = form.cleaned_data['rz_lgr']
-        ctx['lgr_id'] = rz_lgr  # needed to download results as csv
```

Comment:
@j-bernard thanks to check if this is correct. I don't know exactly this csv is refering to...

Reply:
you need to check lgr_validate_csv url, lgr_id is used here